pkgs/test/cross/default.nix

```diff
@@ -133,6 +133,7 @@ let
     pkgs.pkgsMusl.stdenv
     pkgs.pkgsLLVM.stdenv
     pkgs.pkgsStatic.bash
+    pkgs.pkgsCross.powernv.linux_latest
```

Comment:
IMO the kernel is too heavy to be here because it takes a massive amount of memory to build, and the kernel is not really “cross sanity”

Reply:
Note: these expressions are not built automatically by anything, anywhere (not Hydra, not OfBorg).  It's an (executable) list of reminders for people who are doing large refactorings, so they can easily check that they aren't breaking things they personally don't use often.

Shall I split `sanity` into `sanity-small` and `sanity-full`?

The kernel builds need to be in here somewhere.  The kernel has totally bespoke and different build processes for different architectures (yuck, I know).

Reply:
When imagining sanity one thinks of sane compilers or build environments 

How about a separate set for the kernels? `tests.cross.kernels`